aexpect/client.py

```diff
@@ -187,7 +187,7 @@ def __init__(self, command=None, a_id=None, auto_close=False, echo=False,
         self.reader_fds = {}
         try:
             assert(is_file_locked(self.lock_server_running_filename))
-            for reader, filename in self.reader_filenames.items():
+            for reader, filename in list(self.reader_filenames.items()):
```

Comment:
Why is this needed I don't see the reason for turning the `dict_items` (in Python 3) into a `list`. The former even performs better being working as an iterator.

Can you elaborate on the reason for this change?

Reply:
sorry, I was just think this will work for both python 2 and 3. But I was forgetting it is compatible.
Will close this patch. Thanks.